themes/scalereal/static/css/case-study.css

```diff
@@ -43,15 +43,17 @@
 }
 .case-study-page img{
   width: 100%;
+  padding: 0;
   margin: 0;
+  max-width: 100%;object-fit: contain;
```

Comment:
```suggestion
  max-width: 100%;
  object-fit: contain;
```

Reply:
changes made in commit [39e255f](https://github.com/scalereal/website/pull/162/commits/39e255fdbb314471715c105a758626c3873ef95d)